code/addons/vitest/src/components/TestProviderRender.tsx

```diff
@@ -319,63 +318,66 @@ export const TestProviderRender: FC<TestProviderRenderProps> = ({
                 />
               }
             />
-            <WithTooltip
-              hasChrome={false}
-              trigger="hover"
-              tooltip={
-                <TooltipNote
-                  note={
-                    watching
-                      ? 'Unavailable in watch mode'
-                      : currentRun.triggeredBy &&
-                          !FULL_RUN_TRIGGERS.includes(currentRun.triggeredBy)
-                        ? 'Unavailable when running focused tests'
-                        : isRunning
-                          ? 'Testing in progress'
-                          : currentRun.coverageSummary
-                            ? 'View coverage report'
-                            : fatalError
-                              ? 'Component tests crashed'
-                              : 'Run tests to calculate coverage'
-                  }
-                />
-              }
-            >
-              {watching ||
-              (currentRun.triggeredBy && !FULL_RUN_TRIGGERS.includes(currentRun.triggeredBy)) ? (
-                <IconButton size="medium" disabled>
-                  <InfoIcon
-                    aria-label={
-                      watching
-                        ? `Coverage is unavailable in watch mode`
-                        : `Coverage is unavailable when running focused tests`
-                    }
-                  />
-                </IconButton>
-              ) : currentRun.coverageSummary ? (
-                <IconButton asChild size="medium">
-                  <a href="/coverage/index.html" target="_blank" aria-label="Open coverage report">
-                    <TestStatusIcon
-                      isRunning={isRunning}
-                      percentage={currentRun.coverageSummary.percentage}
-                      status={currentRun.coverageSummary.status}
-                      aria-label={`Coverage status: ${currentRun.coverageSummary.status}`}
-                    />
-                    <span aria-label={`${currentRun.coverageSummary.percentage} percent coverage`}>
-                      {currentRun.coverageSummary.percentage}%
-                    </span>
-                  </a>
-                </IconButton>
-              ) : (
-                <IconButton size="medium" disabled>
+
+            {/* FIXME: aria labels were not 100% consistent with the tooltip logic. Double check this logic during review please! */}
```

Comment:
@ghengeveld @yannbf I need input here to understand if there also are inconsistencies between how the testing module works and how the original UI reported its state.

Also, see below, there's a conditional branch that I think is unreachable in practice but I'd need confirmation if possible.

Thanks

Reply:
Makes sense, I'm curious if this exists for the sake of showing stale coverage reports while tests are running (esp. in watch mode when tests might be running most of the time). I'll leave the JS comments, I'm writing an issue for us to review logic around tooltips/labels in the entire component later.